contracts/exchange/test/internal.ts

```diff
@@ -439,5 +2357,240 @@ blockchainTests('Exchange core internal functions', env => {
             expect(logs[3].args.amount).to.bignumber.eq(fillResults.makerFeePaid);
         });
     });
+
+    blockchainTests('settleMatchOrders', () => {
+        const getLeftOrder = () => {
```

Comment:
Why have a separate left and right order generators? I don't think `_settleMatchOrders()` cares if the orders are complementary.

Reply:
You're right. I did it to make the tests seem more realistic, but I'll change it